### PR TITLE
change loading to accommodate twist loading

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by mjburton11** [comment]
in conjunction with convexengineering/gplibrary#147

File: solar/sankey_visualization.ipynb [hunk 1]
```diff
@@ -0,0 +1,288 @@
+{
```
Comment (bqpd) [inline solar/sankey_visualization.ipynb]: 288 lines of sankey ipynb on the wall, 288 lines of sankey,,,

**Comment by mjburton11** [comment]
test this please

File: solar/sankey_visualization.ipynb (hunk 1 above)
Comment (mjburton11) [reply solar/sankey_visualization.ipynb]: That's annoying.  How do I get rid of this?

**Comment by bqpd** [comment]
Unfortunately it's just how ipython notebooks are...

**Pull request merged**